target/linux/ramips/dts/mt7621_netgear_r6800.dts

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: GPL-2.0
+/dts-v1/;
+
+#include "mt7621_netgear_sercomm_bzv.dtsi"
+
+/ {
+	compatible = "netgear,r6800", "mediatek,mt7621-soc";
+	model = "Netgear R6800";
+};
+
+&led_wps {
+	label = "r6800:white:wps";
+};
```

Comment:
Since there is only one line per LED left in the dtsi and we need all those node labels, I would strongly prefer having the whole LED block in the DTS file.

Reply:
Will be done.

Reply:
BTW: How probable is it that we will receive other devices using the DTSI soon? I just wonder whether it wouldn't be easier to put everything into DTS and create DTSI when we have a second device that needs it.

Reply:
[Techinfodepot](http://en.techinfodepot.shoutwiki.com/wiki/Netgear_R6800) says, that the same board is used in 5 more devices. But I don't have other boards for tests. 

It is possible to add support for all but I'm not sure when it will happen.